### PR TITLE
clarify modern authentication default value

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #489

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ For the Office 365 services, the default state of modern authentication is:
 - Turned **on** for SharePoint Online by default. 
     
 - Turned **on** for Skype for Business Online by default. See [Enable Skype for Business Online for modern authentication ](https://social.technet.microsoft.com/wiki/contents/articles/34339.skype-for-business-online-enable-your-tenant-for-modern-authentication.aspx)to turn it off or on.
+
+> [!NOTE]
+> For tenants created **before** August 1, 2017, modern authentication is **off** by default for Exchange Online and Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md]:
```suggestion
> For tenants created **before** August 1, 2017, modern authentication is turned **off** by default for Exchange Online and Skype for Business Online.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9, your recommendation was successfully applied
Thank you

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #489.

Thank you.

**Comment by MSFTTracyP** [comment]
Looking at this PR now.

**Pull request merged**